base/applications/mspaint/dib.cpp

```diff
@@ -333,3 +333,61 @@ HBITMAP SkewDIB(HDC hDC1, HBITMAP hbm, INT nDegree, BOOL bVertical)
     DeleteDC(hDC2);
     return hbmNew;
 }
+
+typedef struct tagBITMAPINFODX
+{
+    BITMAPINFOHEADER bmiHeader;
+    RGBQUAD          bmiColors[256];
+} BITMAPINFODX, *LPBITMAPINFODX;
+
+HGLOBAL BitmapToClipboardDIB(HBITMAP hBitmap)
```

Comment:
Does this support monochrome or 16-color mode images that have been opened in mspaint, and are then copied in parts to the clipboard?

Reply:
16-bit color DIB is not supported. it is rarely used.

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/7851a3e6-a6e0-4ff2-93c4-6e850294f38e)